L1Trigger/CSCTriggerPrimitives/interface/CSCCathodeLCTProcessor.h

```diff
@@ -162,6 +168,12 @@ class CSCCathodeLCTProcessor : public CSCBaseboard {
   void dumpDigis(
       const std::vector<int> strip[CSCConstants::NUM_LAYERS][CSCConstants::MAX_NUM_HALF_STRIPS_RUN2_TRIGGER]) const;
 
+  /* check whether there is a shower around best CLCT */
+  void checkLocalShower(
+      int zone, const std::vector<int> strip[CSCConstants::NUM_LAYERS][CSCConstants::MAX_NUM_HALF_STRIPS_RUN2_TRIGGER]);
```

Comment:
It is cheaper to pass the vector by reference:
```suggestion
  void checkLocalShower(
      int zone, const std::vector<int>& strip[CSCConstants::NUM_LAYERS][CSCConstants::MAX_NUM_HALF_STRIPS_RUN2_TRIGGER]);
```

Reply:
i tried to switch to vector reference in checkLocalShower, but since this vector is used in pulseExtension, the compiling failed... Multiple places in ths CSC trigger emulator used the vector rather reference as argument. I think it is better to do a individual commit to improve this.